Cognite.Simulator.Utils/ConfigurationLibraryBase.cs

```diff
@@ -279,8 +273,9 @@ private async Task ReadConfigurations(CancellationToken token)
                 {
                     Filter = new SimulatorRoutineRevisionFilter()
                     {
-                        // TODO filter by created time, simulatorExternalIds, simulatorIntegrationExternalIds
+                        // TODO filter by created time, simulatorIntegrationExternalIds
```

Comment:
question(non-blocking): Why do we need to filter by created time?

Reply:
we don't really have to, but this is a way to limit the amount of resources needed to be downloaded from the API every time. (i.e. don't have to refetch the revision that was already fetched yesterday)
This is not currently implemented in the new routines logic though...

Reply:
It used to be done with the file-powered APIs